src/components/Card/PCard.vue

```diff
@@ -14,5 +26,20 @@
   p-6
   rounded-lg
   shadow-sm
+  transition-all;
+
+  /*
+    Explicitly defining these after transition-all
+      to override the transition-property on the box-shadow
+      because transitioning shadows isn't super performant
+      and it doesn't add much value.
+  */
```

Comment:
Personally I don't think the comment is needed. We do this in a few places where we have tailwind and then plain css overrides or extra stuff.